components/mediation-admin/org.wso2.carbon.message.processor/src/main/java/org/wso2/carbon/message/processor/service/MessageProcessorAdminService.java

```diff
@@ -755,4 +761,111 @@ public boolean validateAxis2ClientRepo(String input) throws AxisFault {
         }
     }
 
+    /**
+     * Gets the message from the associated queue.
+     *
+     * @param processorName message processor name.
+     * @return <code>message</code> returns message received from the queue as a string.
+     */
+    public String getMessage(String processorName) throws AxisFault {
```

Comment:
this is more like a peekMessage right? Would it be better to rename the method?

Reply:
Peek message would be more applicable to stacks rather than queues. I named it getMessage since it makes more sense when the front end calls a getMessage function to display. 

If we were to apply a method similar to peek for stacks, then it would be 'browse'. Should I change it to browseMessage() ?

Reply:
I changed getMessage() to browseMessage() 
Leaving getMessageAsString() without any name changes